ably/transitioner_test.go

```diff
@@ -312,6 +373,36 @@ func (c ChanTransitioner) To(path ...ably.ChannelState) (*ably.RealtimeChannel,
 	})
 }
 
+// preserves state context for channel
+func (c *ChanTransitioner) ToSpecifiedState(path ...ably.ChannelState) (*ably.RealtimeChannel, io.Closer) {
```

Comment:
I think earlier version of `to` is already being used at places ..,. we need both for the time being, otherwise we need to refactor the code where earlier version of to is being used. I can rename it to something else for the time being

Reply:
OK, not a blocking issue, can be refactored later.